helmfile/01-applications.yaml

```diff
@@ -419,11 +423,23 @@ releases:
     app: fluentd-aggregator
   chart: stable/fluentd
   version: 2.3.2
-  installed: {{ .Values.fluentd.enabled }}
+  installed: false
```

Comment:
Is the idea here to keep the helm release for a little while so we can uninstall with helmfile and later remove it from the helmfile as well?

Reply:
The original plan with this was to let helmfile automatically uninstall the release, thus mimimizing the manual upgrade steps. But in this case we still have to manually uninstall the release either via helm or helmfile, due to some clashes.
And after we have create a new ck8s release we would then simply remove all entries in helmfile which has `installed: false`

Reply:
Ok. It feels like it will be easy to forget to remove it from helmfile after the next release. But I guess someone will remember it or notice it after a while.
An alternative is to remove it now and uninstall with helm instead of helmfile.

Reply:
As we have a couple of other releases with `installed: false` i'll keep it the way it is right now.
I quickly created https://github.com/elastisys/compliantkubernetes-apps/issues/155 just to have it somehwere to remeber to remove these releases.